samples/dsm/akeneo-dsm/front-end/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
```

Comment:
Not sure about this as this is a dockerised project. Either change to "In the container dsm-front" or change the command to a docker-compose run / make command one.

Reply:
There is a README task ODX-287 in the next sprint to fix all the README.